src/menu/MenuSongSelection.as

```diff
@@ -803,7 +807,7 @@ package menu
                 }
                 else
                 {
-                    if (options.infoTab == TAB_PLAYLIST)
+                    if (tarSongItem.isLocked == false && options.infoTab == TAB_PLAYLIST)
```

Comment:
Can this just be `!tarSongItem.isLocked` instead of `tarSongItem.isLocked == false` for consistency sake?

Reply:
Changed.